apps/keyboard/js/render.js

```diff
@@ -115,7 +91,8 @@ var IMERender = (function() {
     } else {
       var options = {
         classNames: ['keyboard-type-container'],
-        totalWidth: ime.clientWidth
+        totalWidth: ime.clientWidth,
```

Comment:
I wonder if we could remove the need for `totalWidth` in the future -- please file a follow-up.